mne/export/_egimff.py

```diff
@@ -40,6 +42,7 @@ def export_evokeds_mff(fname, evoked, history=None, *, overwrite=False,
     Notes
     -----
     .. versionadded:: 0.24
+    %(export_warning_note_evoked)s
```

Comment:
Not sure if it's here or above, but:
```
E   mne.export._egimff.export_evokeds_mff : GL03 : Double line break found; please use only one blank line to separate sections or paragraphs, and do not leave blank lines at the end of docstrings
```

Reply:
Yes, same as before this additional change... I'm sorry, but I don't get why this one is being raised. Is it because the docstring starts with a 'summary' line and does not have a 'description' before the warning?

Reply:
the multi-line string `export_warning_note` starts and ends with a newline.  Probably you need to remove one or both of those.  I'd guess it's the trailing newline.

Reply:
Looks like that's not it.. or I missed something.